### PR TITLE
Flesh out README to help out future contributors

### PR DESCRIPTION
**Pull request by sbrl**
The README was looking a bit bare, so I've taken the time to flesh it out a bit.

This PR was submitted from Freeside Desktop #2!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,30 @@
...
+
+Run a task:
+```bash
+fab -R {desktops,servers} taskname` 
+```
 
 Example:
-`fab -R desktops install_neovim` 
+```bash
+fab -R desktops install_neovim`
```
Comment (Epictek) [inline README.md]: no need for ` at the end

File: README.md [hunk 2]
```diff
@@ -1,9 +1,30 @@
...
+```
+
+## Contributing
+Want some software installing on the Freeside computers? You've come to the right place! Here's a simnple guide on how to request software to be installed:
+
+1. Make sure you're logged into your GitHub account.
+2. Take a look at [fabfile/__init__.py](https://github.com/FreesideHull/Freeside-Fabfiles/blob/master/fabfile/__init__.py). 
+3. Press the edit button in the top-right.
+4. Figure out how to install the software you want on a standard _Fedora_ machine.
+5. Add a new function to the _Fabfile_ in the same vein that the existing ones that installs the software you want. We prefer it if the software is available through `dnf`, _Fedora_'s official package manager. THat way it stays up-to-date automatically!
```
Comment (Epictek) [inline README.md]: Typo at "THat way"

**Review by Epictek — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sbrl) [reply README.md]: oops!

**Review by Epictek — APPROVED** [review]

**Comment by sbrl** [comment]
Many thanks for spotting those errors, @Epictek :D

**Comment by Epictek** [comment]
[LGTM](https://www.lgtm.in/i/0uNmb71vn)

[:+1:](https://www.lgtm.in/u/0uNmb71vn) [:-1:](https://www.lgtm.in/r/0uNmb71vn)

**Pull request merged**